css/src/metabox.css

```diff
@@ -596,7 +596,13 @@
 
 ul.yoast-seo-social-share-buttons li {
 	display: inline-block;
-	margin-right: 16px;
+	margin-right: 24px;
+}
+
+ul.yoast-seo-social-share-buttons li .x-share svg{
+	width: 30px;
+	height: 30px;
```

Comment:
Yeah, we checked with @uxkai and we had to make the X logo a bit smaller, because of the shape of it. Otherwise, it looked bigger than the FB one, even though they shared the same dimensions